lib/express_validator.js

```diff
@@ -165,6 +171,29 @@ var expressValidator = function(options) {
 };
 
 /**
+ * validate an object using a schema
+ *
+ * @method validateParams
+ * @param  {Object}       schema    schema of validations
```

Comment:
I know that this is described in the docs and tests, but it would be nice if the schema was described in detail here, so that it is 100% clear what is expected/allowed.